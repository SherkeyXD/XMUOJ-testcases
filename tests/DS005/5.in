734
224 0 204 0 0 165 232 0 0 0 0 65 30 299 176 0 0 0 0 0 0 0 0 96 78 196 60 240 165 204 154 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 87 132 242 122 249 204 225 234 50 1 27 7 28 294 168 18 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 212 92 174 160 50 159 236 129 159 134 6 249 120 220 276 132 104 4 15 48 200 210 287 240 268 119 82 120 282 172 60 81 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 129 221 28 9 196 45 138 204 274 135 72 183 50 61 189 50 192 100 162 134 236 18 0 60 240 204 0 258 194 60 250 78 128 2 276 228 62 48 96 140 272 0 204 150 30 60 259 36 268 25 268 0 222 255 200 110 192 92 120 156 184 76 224 132 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 260 156 95 46 286 260 83 36 176 288 270 1 218 7 206 168 100 168 33 118 96 140 120 172 216 180 80 294 170 296 132 234 86 216 96 176 40 260 285 112 15 57 30 169 0 0 258 6 90 200 270 128 0 0 270 84 246 180 80 250 24 274 210 84 37 65 39 263 146 65 101 275 208 257 147 273 144 36 80 42 50 164 0 0 244 294 260 64 66 284 95 150 259 210 296 105 110 75 57 264 272 97 0 0 16 150 57 125 56 88 171 144 228 223 240 204 294 252 250 294 210 99 88 96 122 6 282 228 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 
545 558
