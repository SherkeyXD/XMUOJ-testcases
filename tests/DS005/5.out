ERROR: T[545] is NULL
